cmd/zstream/zstream_decompress.c

```diff
@@ -71,8 +72,26 @@ zstream_do_decompress(int argc, char *argv[])
 
 	if (hcreate(argc) == 0)
 		errx(1, "hcreate");
-	for(i = 0; i < argc; i++) {
-		ENTRY e = {.key = argv[i]};
+	for (i = 1; i < argc; i++) {
+		int64_t inode, offset;
+		char *ino_str;
+		char *key;
+
+		key = malloc(KEYSIZE);
+
+		ino_str = strsep(&argv[i], ",");
+		if (argv[i] == NULL) {
+			zstream_usage();
+			exit(2);
+		}
+		errno = 0;
+		inode = strtoul(ino_str, NULL, 0);
+		offset = strtoul(argv[i], NULL, 0);
```

Comment:
`ull`, they're both 64 bits (and unsigned, which you're losing; if not here, then in the format)

Reply:
I did it this way so I could more easily detect if the user supplied a negative number.  `stroul` will happy accept a negative number, and testing for negativity is easier than comparing to `LONG_MAX`.  Would you rather I change it?